components/Slider/examples/SliderAutolayoutExampleViewController.m

```diff
@@ -33,26 +34,27 @@ - (id)init {
   self = [super init];
   if (self) {
     self.colorScheme = [[MDCSemanticColorScheme alloc] init];
+    self.typographyScheme = [[MDCTypographyScheme alloc] init];
   }
   return self;
 }
 
 - (void)viewDidLoad {
   [super viewDidLoad];
-  // Do any additional setup after loading the view, typically from a nib.
 
   // Ensure a consistent starting state
   _materialSlider.enabled = YES;
   _vanillaSlider.enabled = _materialSlider.enabled;
   _enabledSwitch.on = _materialSlider.enabled;
   _materialSlider.value = _vanillaSlider.value;
-  
-  self.materialSlider.statefulAPIEnabled = YES;
-  
+    
   [MDCSliderColorThemer applySemanticColorScheme:self.colorScheme toSlider:self.materialSlider];
+
+  _materialLabel.font = self.typographyScheme.body1;
+  _uiLabel.font = self.typographyScheme.body1;
 }
 
-- (IBAction)materialSliderDidChange:(id)sender {
+- (IBAction)mdcSliderDidChange:(id)sender {
```

Comment:
If the property is called `materialSlider`, why did we rename the method?

Reply:
I was trying to track down the bug and remade the storyboard, this must have got changed in doing that.